src/lib/modules/ens/index.js

```diff
@@ -233,14 +234,30 @@ class ENS {
             return eachCb(error);
           }
 
+          const directives = directivesRegExp.exec(address);
+          if (directives && directives.length) {
+            this.embark.registerActionForEvent("contracts:deploy:afterAll", async (deployActionCb) => {
+              this.events.request("contracts:contract", directives[1], (contract) => {
+                if(!contract) return deployActionCb(); // if the contract is not registered in the config, it will be undefined here
```

Comment:
You should print a warning so that the user knows that he tried to register something to an address we don't know about.